nilearn/plotting/displays/_slicers.py

```diff
@@ -461,14 +534,23 @@ def _map_show(
             )
 
         data_2d_list = []
+        transparency_list = []
         for display_ax in self.axes.values():
             try:
                 data_2d = display_ax.transform_to_2d(data, affine)
             except IndexError:
                 # We are cutting outside the indices of the data
                 data_2d = None
 
+            if isinstance(transparency, (np.ndarray)):
+                transparency_2d = display_ax.transform_to_2d(
+                    transparency, transparency_affine
+                )
```

Comment:
seems impossible for now to get this to work with anything but single scalar value for alpha value

need to check if there is some weird interaction when using imshow with masked arrays

Reply:
OK this is strange: it works fine in the doc build but not locally...

![image](https://github.com/user-attachments/assets/bc78a07d-b978-4ac9-8281-429ad7c77bc4)


I have seen some issues linked to alpha layers in matplolib so maybe older versions (the doc build uses 3.3) work better.

Reply:
This seems to be related to this one: https://github.com/matplotlib/matplotlib/issues/26092